meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -772,22 +772,28 @@ DISTRO_FEATURES:append:tegra = " opengl"
 DISTRO_FEATURES_DEFAULT:remove:tegra = "modsign"
 PREFERRED_PROVIDER_virtual/kernel:tegra ?= "linux-tegra"
 IMAGE_TEGRAFLASH_FS_TYPE:sota:tegra = "${@bb.utils.contains('TEGRA_EXTERNAL_ROOTFS_WIC', '1', 'wic.gz', 'ota-ext4', d)}"
+IMAGE_TEGRAFLASH_ESPIMG:sota:tegra = "${IMGDEPLOYDIR}/${IMAGE_LINK_NAME}.ota-esp"
 IMAGE_FSTYPES:remove:tegra = "${@bb.utils.contains('TEGRA_EXTERNAL_ROOTFS_WIC', '1', '', 'wic wic.gz wic.bmap', d)}"
+TEGRA_ESP_IMAGE:sota:tegra = ""
 TEGRA_INITRAMFS_FSTYPES:tegra = ""
 INITRD_IMAGE:tegra = ""
 IMAGE_TEGRAFLASH_KERNEL:tegra = "${DEPLOY_DIR_IMAGE}/${KERNEL_IMAGETYPE}"
 WKS_FILE:tegra:sota ?= "efidisk-sota.wks.in"
+IMAGE_EFI_BOOT_FILES:sota:tegra ?= "${@make_efi_dtb_boot_files(d)}"
```

Comment:
Wonder if we need the dtb boot files here.

Reply:
not really, I've removed the line